data/lsmfip

```diff
@@ -292,7 +292,7 @@ if (!$options{verify}) {
                 print "$n\n";
             }
             else {
-                die "$n was requested but nothing was installed";
+                warn "$n was requested but nothing was installed";
```

Comment:
Why warn, will this not just hide when there is issue ?

Reply:
It will reach this line only in case all package to be installed are "new".

Reply:
instead of `warn` record_info {type fail}? or a soft fail alltogether?

Reply:
Let me try!